### PR TITLE
Created tokens registry canister

### PR DESCRIPTION
**Pull request by miguelToscano**

### CODE REVIEW

File: candid/tokens.did [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+};
+
+service : {
+    "name" : () -> (text) query;
+    "get_all"      : () -> (vec token) query;
+
+    "add"    : (input_add_token) -> (operation_response);
+    "edit"   : (input_edit_token) -> (operation_response);
+    "remove" : (text) -> (operation_response);
+    "set_controller" : (principal) -> (operation_response);
```
Comment (jsonsivar) [inline candid/tokens.did:56]: We should add get by canister id/name asap.

**Review by jsonsivar — APPROVED** [review]
LGTM. Let's leave this PR open until @Nima-Ra reviews and also the README is updated (@miguelToscano @nicolaspoggi ). We will deploy this version and smoke test now.

File: registries/tokens/src/tokens.rs [hunk 2]
```diff
@@ -0,0 +1,492 @@
+use ic_cdk::export::candid::{CandidType, Principal};
```
Comment (Nima-Ra) [inline registries/tokens/src/tokens.rs:1]: CandidType and Principal should be imported from ic_kit.

File: registries/tokens/src/upgrade.rs [hunk 3]
```diff
@@ -0,0 +1,37 @@
+use crate::tokens::{Controller, Token, TokenRegistry};
+use ic_cdk::export::candid::{CandidType, Deserialize, Principal};
```
Comment (Nima-Ra) [inline registries/tokens/src/upgrade.rs:2]: CandidType, Deserialize and Principal should be imported from ic_kit.

**Review by Nima-Ra — CHANGES_REQUESTED** [review]
Good job! A few changes are needed though.
I think you should run `cargo fmt` as well.

**Review by Nima-Ra — APPROVED** [review]

**Pull request merged**